include/gatekeeper_gk.h

```diff
@@ -115,6 +115,8 @@ struct gk_instance {
 	struct token_bucket_ratelimit_state front_icmp_rs;
 	struct token_bucket_ratelimit_state back_icmp_rs;
 	unsigned int num_scan_del;
+	int num_policies;
+	struct ggu_policy **policies;
```

Comment:
Adding these fields here are complicating the code. Moving arrays `policies` and `flow_hash_val_arr` from `add_ggu_policy_bulk()` to `process_cmds_from_mailbox()` would eliminate the need to have these fields here. `process_gk_cmd()` would add the entries to arrays `policies` and `flow_hash_val_arr` as needed.